dev-docs/source/VisualStudioBuildImpact.rst

```diff
@@ -0,0 +1,76 @@
+Visual Studio Build Impact
+==========================
+
+Building Mantid on Visual Studio can take a while, and really tie up the
+computer. This is because Visual Studio starts all of the compilation
+processes at normal priority, so they compete as equals with everything
+you are doing.
+
+So you might think that you can open task manage and reduce the priority
+of all the cl.exe processes, well there can be quite a few, and no it
+won't work for long as these processes are replaced for every file it
+compiles.
+
+What you want to do it hunt down the MSBuild.exe processes and reduce
+the priority of them, there should be the same number as you have
+logical processors. The MSBuild processes spawn all of the compiler and
+linker tasks, an they inherit the priority of the MSBuild process.
+
+Script
+------
+
+Of course if you don't want to do this yourself then you can use this
+script.
+
+.. raw:: html
```

Comment:
Was this translated with pandoc? I think it has become confused here and the output is odd as we shouldn't have raw html here. I think a literal block is probably okay here?